tests/test_browser.py

```diff
@@ -3858,6 +3858,11 @@ def test_webgl_from_client_side_memory_without_default_enabled_extensions(self):
   def test_webgl_offscreen_framebuffer(self):
     self.btest('webgl_draw_triangle.c', '0', args=['-lGL', '-s', 'OFFSCREEN_FRAMEBUFFER=1', '-DEXPLICIT_SWAP=1'])
 
+  # Tests that -s WORKAROUND_OLD_WEBGL_UNIFORM_UPLOAD_IGNORED_OFFSET_BUG=1 rendering works.
+  @requires_graphics_hardware
```

Comment:
Does this actually test that code path - does it force the polyfill on somehow?

Reply:
It just tests that code compiles with that flag enabled. I don't think it is worth spending time to maintaining a test infrastructure where we would enforce the path to kick in.